ui/app/AppLayouts/Chat/controls/community/InDropdown.qml

```diff
@@ -0,0 +1,368 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Popups 0.1
+
+import shared.controls 1.0
+
+import SortFilterProxyModel 0.2
+
+StatusDropdown {
+    id: root
+
+    property string communityName
+    property string communityImage
+    property color communityColor
+
+    property alias model: topLevelModel.sourceModel
+
+    width: 289
+    padding: 8
+
+    // force keeping within the bounds of the enclosing window
+    margins: 0
+
+    signal addChannelClicked
+    signal communitySelected
+    signal channelsSelected(var channels)
+
+    function setSelectedChannels(channels) {
+        d.setSelectedChannels(channels)
+    }
+
+    QtObject {
+        id: d
+
+        function search(text, searcherText) {
+            return text.toLowerCase().includes(searcherText.toLowerCase())
+        }
+
+        function resolveEmoji(emoji) {
+            return !!emoji ? emoji : ""
+        }
+
+        function resolveColor(color, colorId) {
+            return !!color ? color : Theme.palette.userCustomizationColors[colorId]
+        }
+
+        signal setSelectedChannels(var channels)
+
+        readonly property var selectedChannels: new Map()
+
+        function addToSelectedChannels(model) {
+            selectedChannels.set(model.itemId, {
+                itemId: model.itemId,
+                name: model.name,
+                color: d.resolveColor(model.color, model.colorId),
+                emoji: d.resolveEmoji(model.emoji)
+            })
+        }
+
+        function removeFromSelectedChannels(model) {
+            selectedChannels.delete(model.itemId)
+        }
+    }
+
+    ColumnLayout {
+        anchors.top: parent.top
+        anchors.right: parent.right
+        anchors.left: parent.left
+
+        spacing: 0
+
+        SearchBox {
+            id: searcher
+
+            Layout.fillWidth: true
+
+            topPadding: 0
+            bottomPadding: 0
+            minimumHeight: 36
+            maximumHeight: 36
+        }
+
+        StatusListItem {
+            Layout.fillWidth: true
+            Layout.topMargin: 9
+
```

Comment:
same here

Reply:
Here it's not beneficial bc of radio button, overriding `components` property won't remove check box from base. For simplicity I would leave it as it is.